src/providers/CompletionItemProvider.ts

```diff
@@ -180,10 +180,13 @@ const getPythonSymbols = async (isFunction: boolean, lineText: string, position:
   const modules = Object.values(modulesByUri);
   modules.push(l10n.t("New module name"));
   cis.push(
-    getCompletionItemInString(isFunction ? l10n.t("create a new function") : l10n.t("create a new class"), lineText, position, [
-      modules.length === 1 ? modules[0] : modules,
-      isFunction ? l10n.t("function name") : l10n.t("class name"),
-    ], getPythonSuffix(isFunction))
+    getCompletionItemInString(
+      isFunction ? l10n.t("create a new function") : l10n.t("create a new class"),
```

Comment:
Is all lower case ok in the context?

Reply:
it's the last option of the completion list ...

Reply:
![image](https://user-images.githubusercontent.com/90181748/213254149-ef0186f4-a1d5-4098-8434-1468365a8ecc.png)

Reply:
What do you think ?